src/styles/main-block.scss

```
@@ -0,0 +1,26 @@
+.main-block {
+  display: flex;
+  justify-content: center;
+
+  &__cards {
+    display: flex;
+    justify-content: center;
+    flex-wrap: wrap;
+    max-width: 950px;
+    padding: 50px 0;
+  }
+
+  &__item {
+    margin-right: 48px;
+    margin-bottom: 46px;
+  }
+
+  &__item:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  
+  &__item:nth-child(n + 5) {
```

```suggestion
  }
  
  &__item:nth-child(n + 5) {
```
Одной линии отступа обычно достаточно